File: Code/Client_Prototype/Client_Prototype/Classes/Abteilung.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Windows;
using BSD_Client.Classes;

namespace BSD_Client
{
    public class Abteilung
    {
        public int ab_id { get; set; }
        public string ab_name { get; set; }
        public int ab_etage { get; set; }
        public List<Stand> ab_stande = new List<Stand>();
        //public Polygon shape { get; set; }
        public Quiz ab_quiz { get; set; }

        public Abteilung()
        {
            
        }

        public Abteilung(int _ID, String _Name, int _Etage)
        {
            ab_id = _ID;
            ab_name = _Name;
            ab_etage = _Etage;
        }

        public void addStandToAbteilung(Stand _Stand)
        {
            ab_stande.Add(_Stand);
        }

        public override String ToString()
        {
            return "Name: "+this.ab_name+"; ID:"+this.ab_id+"; Etage: "+this.ab_etage+"; Stands:"+this.ab_stande;
        }


    }
}

File: Code/Android-Project/Tatue-Rater2/app/src/main/java/com/htl_villach/tatue_rater/Classes/Abteilung.java
package com.htl_villach.tatue_rater.Classes;

import java.util.ArrayList;
import java.util.List;

/**
 * Created by simon on 10.03.2016.
 */
public class Abteilung {
    public int ab_id;
    public String ab_name;
    public int ab_etage;
    public List<Stand> ab_staende = new ArrayList<Stand>();
    public Polygon shap;
    public Quiz ab_quiz;

    public Abteilung() {
    }

    public Abteilung(int ab_id, String ab_name, int ab_etage) {
        this.ab_id = ab_id;
        this.ab_name = ab_name;
        this.ab_etage = ab_etage;
    }

    public int getAb_id() {
        return ab_id;
    }

    public void setAb_id(int ab_id) {
        this.ab_id = ab_id;
    }

    public String getAb_name() {
        return ab_name;
    }

    public void setAb_name(String ab_name) {
        this.ab_name = ab_name;
    }

    public int getAb_etage() {
        return ab_etage;
    }

    public void setAb_etage(int ab_etage) {
        this.ab_etage = ab_etage;
    }

    public List<Stand> getAb_stande() {
        return ab_staende;
    }

    public void setAb_stande(List<Stand> ab_stande) {
        this.ab_staende = ab_stande;
    }

    public Polygon getShap() {
        return shap;
    }

    public void setShap(Polygon shap) {
        this.shap = shap;
    }

    public Quiz getAb_quiz() {
        return ab_quiz;
    }

    public void setAb_quiz(Quiz ab_quiz) {
        this.ab_quiz = ab_quiz;
    }

    @Override
    public String toString() {
        return "Abteilung{" +
                "ab_id=" + ab_id +
                ", ab_name='" + ab_name + '\'' +
                ", ab_etage=" + ab_etage +
                ", ab_stande=" + ab_staende +
                ", shap=" + shap +
                ", ab_quiz=" + ab_quiz +
                '}';
    }
}
